pkgs/development/python-modules/tinyprog/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k
+, intelhex
+, jsonmerge
+, packaging  
+, pyserial
+, pyusb
+, six
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "tinyprog";
+  version = "1.0.21";
+  format = "wheel";
+
+  src = fetchPypi ({
```

Comment:
Avoid using wheels. Instead, fetch from GitHub
https://github.com/tinyfpga/TinyFPGA-Bootloader/tree/master/programmer
Note the source is in a subfolder so `sourceRoot` needs to be set.